### PR TITLE
merging accept or decline invite buttons

### PR DESCRIPTION
**Pull request by LoganTown**
For dashboard view, added attending invites to the all events filter

For event_home, added a testing list for attendees and invitees, then buttons to accept or decline a invite if your in the invitees list

for models, I believe I added attendees stuff to profile and events

for views, changed event_home, and added a handle_event view that deals with changing the attendees and invitees list.

### CODE REVIEW

**Comment by LoganTown** [comment]
I am not sure why theres 2 other commits that want merged to main when i made this request, they shouldnt cause issues but if anyone knows why they are there please let me know

File: our_site/planner/templates/planner/event_home.html [hunk 1]
```diff
@@ -29,6 +31,36 @@ <h1 class="title is-1">{{ event.name }}</h1>
...
+            </div>
+        </div>
+    </div>
+
+
+
+    {% if user.profile in invitees %}
+        <form method="post" action="{% url 'planner:handle_event' event.id %}">
+            {% csrf_token %}
+            <button type="submit" name="action" value="accept">Accept Invite</button>
```
Comment (mnugent04) [inline our_site/planner/templates/planner/event_home.html:60]: not needed but would be nice if the buttons had the same css as all the others just so it looks uniform

File: our_site/planner/urls.py [hunk 2]
```diff
@@ -8,5 +8,6 @@
     path("<int:event_id>/edit_event/", views.edit_event, name="edit_event"),
     path('event/<int:event_id>/', views.event_home, name='event_home'),
     path("<int:event_id>/invitations/", views.invitations, name="invitations"),
-    path('addRoles/', views.addRoles, name="addRoles")
+    path('addRoles/', views.addRoles, name="addRoles"),
+    path('<int:event_id>/handle_event/', views.handle_event, name='handle_event'),
```
Comment (mnugent04) [inline our_site/planner/urls.py:12]: we don't have an html page called event_handle? don't we need to have one for this to do anything?

**Review by mnugent04 — CHANGES_REQUESTED** [review]

File: our_site/planner/templates/planner/event_home.html [hunk 3]
```diff
@@ -1,36 +1,94 @@
...
+                    <p> Host: {{ event.owner.user }}</p>
+                    <p> Location: {{ event.location }}</p>
+                    <p> Date: {{ event.date }}</p>
+                </div>
+            </section>
+
+            <div class="container">
+                <div class="columns">
+                    <div class="column is-2">
+                        <form action="{% url 'planner:addRoles' %}">
```
Comment (Curtis-Barnhart) [inline our_site/planner/templates/planner/event_home.html:45]: shouldn't this button be gone?

**Review by Curtis-Barnhart — CHANGES_REQUESTED** [review]
On the dashboard html template, there is no distinction given to whether an event is owned or not owned by the currently logged in user. That and the other comment I made in the review are the only problems I see, everything else looks good.

**Review by mnugent04 — APPROVED** [review]

**Review by Curtis-Barnhart — APPROVED** [review]
I buy it

**Review by Curtis-Barnhart — APPROVED** [review]
I buy it more

**Review by mnugent04 — APPROVED** [review]

**Pull request merged**